### PR TITLE
addressed feedback from #3731 ticket

### PR DESCRIPTION
**Pull request by EMXDigital**
## Type of change
- [x ] Feature

## Description of change
Adding Instream mediatype to EMX Digital. New PR from accidental closed PR [here]. This includes all requested updates to our adapter from that ticket. (https://github.com/prebid/Prebid.js/pull/3731).

### CODE REVIEW

**Comment by jsnellbaker** [comment]
@EMXDigital Thanks for putting the changes together and recreating this PR.  I'll take a look over the updates soon and follow-up.

**Comment by bretg** [comment]
docs PR https://github.com/prebid/prebid.github.io/pull/1252

File: test/spec/modules/emx_digitalBidAdapter_spec.js [hunk 1]
```diff
@@ -4,167 +4,343 @@ import * as utils from 'src/utils';
...
+      //   const testBid = Object.assign({}, bid);
+      //   delete testBid.mediaTypes.video.context;
+      //   expect(spec.isBidRequestValid(testBid)).to.equal(false);
+      // })
+
+      // it('bidRequest with missing video playerSize', function() {
+      //   const testBid = Object.assign({}, bid);
+      //   delete testBid.mediaTypes.video.playerSize;
+      //   expect(spec.isBidRequestValid(testBid)).to.equal(false);
+      // })
```
Comment (jsnellbaker) [inline test/spec/modules/emx_digitalBidAdapter_spec.js]: Are these tests something you want to reactivate?  Or remove?

File: test/spec/modules/emx_digitalBidAdapter_spec.js [hunk 2]
```diff
@@ -4,167 +4,343 @@ import * as utils from 'src/utils';
...
+            }
           }
-        }
-      })).to.equal(true);
+        })).to.equal(true);
+      });
     });
   });
 
+  // describe('emxAdapter helper validity', function () {
```
Comment (jsnellbaker) [inline test/spec/modules/emx_digitalBidAdapter_spec.js]: Same here; should these tests stay or go?

Comment (EMXDigital) [reply test/spec/modules/emx_digitalBidAdapter_spec.js]: We've removed these lines.

File: test/spec/modules/emx_digitalBidAdapter_spec.js (hunk 1 above)
Comment (EMXDigital) [reply test/spec/modules/emx_digitalBidAdapter_spec.js]: Thanks, we've removed these lines in our latest commit.

File: test/spec/modules/emx_digitalBidAdapter_spec.js [hunk 3]
```diff
@@ -4,167 +4,241 @@ import * as utils from 'src/utils';
...
+          [300, 600]
+        ],
+        'bidId': '30b31c2501de1e',
+        'bidderRequestId': '22edbae3120bf6',
+        'auctionId': '1d1a01234a475'
+      };
 
-    it('should return true when required params found', function () {
-      expect(spec.isBidRequestValid(bid)).to.equal(true);
+      // it('should return true when required params found', function () {
```
Comment (jsnellbaker) [inline test/spec/modules/emx_digitalBidAdapter_spec.js]: @EMXDigital It seems there were some other commented unit tests.  Can you remove these as well?

Comment (EMXDigital) [reply test/spec/modules/emx_digitalBidAdapter_spec.js]: Apologies for missing this, it is also removed.

**Comment by jsnellbaker** [comment]
@EMXDigital Thanks for making the extra updates; LGTM

**Pull request merged**